### DISCUSSION TITLE
reload_data.py 的路徑問題請教

### DISCUSSION THREAD

**Discussion by yunchiahsu** [yzyouzhang/AIR-ASVspoof#43] [Q&A]
Hi,
我想請教reload_data.py這個程式的問題，在這個程式中的
`    for i in range(len(matfiles)):`    
        `    if matfiles[i][len(path_to_mat)+len(part)+1:].startswith('LFCC'):`    
           `   key = matfiles[i][len(path_to_mat) + len(part) + 6:-4]`
上面這個判斷式是以什麼標準呢?
我把`len(path_to_mat)`打印出來結果是18，這是路徑字串長度
也將`len(part)`打印出來，這是資料集名稱長度

因為我從matlab產生feature後要執行reload_data.py就卡住了，謝謝你。

**Comment by yzyouzhang** [comment]
Hi,
感谢提问！这个key应该是wav的文件名
想问一下你是哪一步卡住了

python版本的LFCC extraction可见https://github.com/yzyouzhang/ASVspoof2021_AIR

谢谢！
